modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -152,22 +193,57 @@ function inope_policy_review_form_submit($form, &$form_state) {
 
   // If 'I have reviewed this policy and approved it for use' is selected.
   if ($form_state['values']['inope_policy_review_status_radio'] == 0) {
-    drupal_set_message(t('Policy approved for use'));
-    // TODO: put this into the database.
+
+    // Only administrators should be able to approve notes.
```

Comment:
See our discussion in chat -- but basically we need to allow staff and admins to answer this in the positive.
The client has not asked for the administrators to change the answers of all staff - so this won't be needed.